packages/runtime/src/runtime.ts

```diff
@@ -29,19 +28,21 @@ export class Runtime {
    */
   private store: State;
   ctx: Context;
-  private appCounter: number;
-  private assetCounter: number;
+  loadedAssetsDefs: ASADefs;
   // https://developer.algorand.org/docs/features/transactions/?query=round
   private round: number;
   private timestamp: number;
-  private readonly loadedAssetsDefs: ASADefs;
 
   constructor (accounts: AccountStoreI[]) {
     // runtime store
     this.store = {
       accounts: new Map<AccountAddress, AccountStoreI>(), // string represents account address
       globalApps: new Map<number, AccountAddress>(), // map of {appId: accountAddress}
-      assetDefs: new Map<number, AccountAddress>() // number represents assetId
+      assetDefs: new Map<number, AccountAddress>(), // number represents assetId
+      assetNameId: new Map<string, number>(),
+      appNameId: new Map<string, number>(),
```

Comment:
maybe we should keep similar names as `deployer` (`sscInfo`, `ASAInfo`)? note: means we return app/asset info/undefined